### ISSUE TITLE
DEMO @ shyiko/typescript-boilerplate leads to 404

### ISSUE THREAD

**Issue by unlight** [shyiko/tsdm#3]
https://github.com/shyiko/typescript-boilerplate

**Comment by shyiko** [comment]
@unlight Yeah, I meant to push it a week ago but got distracted by the stuff at work. Will do in the next couple of days. Thank you for the reminder! (until then I've added "coming soon" notice in the readme.md)
